### PR TITLE
Add page "a.html" with its "a.css"

### PR DESCRIPTION
**Pull request by Sparktan24**
**The following steps were followed:**

- Create a new branch
- Add a .gitignore file.
- Set up [a linter for HTML and CSS and make sure that any files that are not relevant are ignored by git.
- Add descriptive README file to your project.

**Add 3 files (each one in a separate commit):**
- A .html file that includes a header with the text "Hello Microverse!". Use h1 tag for that.
- A .css file that includes one class with styles for your header (make it in your favorite color).
- A test.md file with the text "This file should be ignored by git".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,56 @@
...
+# Hello-World
+
+> Set up a "Hello Microverse" project
+
+
+## Built With
+
+- Html, CSS
+
+## Live Demo (if available)
```
Comment (mgmediaweb) [inline README.md]: Kindly, I suggest you remove the points that you are not using at the moment

File: README.md [hunk 2]
```diff
@@ -1 +1,56 @@
...
+-Have git installed locally.
+-Have a code editor installed (if you wish to modify the project).
+
+### Setup
+-clone the project to a local folder.
+
+
+## Authors
+
+👤 **Author1**
```
Comment (mgmediaweb) [inline README.md]: Kindly, I suggest you put your name here

File: A.html [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="A.css">
+    <title>Document</title>
```
Comment (mgmediaweb) [inline A.html]: Kindly suggest adding a descriptive title of your website.

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @Sparktan24 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost3

## Highlights 💯 
- Professional-looking readme file. ✔️ 
- Clear and descriptive Pull Request. ✔️ 

## Required Changes ♻️

[IMPORTANT] Kindly, I suggest you not close Pull Request, you must update your files in this same PR.

- Kindly, by convention the access file to your page should be called index.html, I suggest you change it

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mgmediaweb — CHANGES_REQUESTED** [review]
Hi @Sparktan24 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost2

 ## Highlights 💯 
- Good job following the advice of the previous reviewer ✔️ 

 ## Required Changes ♻️

[in the previous revision I indicated this]
Kindly, by convention the access file to your page should be called index.html, I suggest you change it

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Sparktan24** [comment]
Hi @mgmediaweb, The instructions of the exercise said to use those names, do I change it anyways?

**Comment by mgmediaweb** [comment]
it is a good practice to do it from the beginning, I recommend doing it

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Sparktan24 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:
- No Linter errors ✅
- Your repo is professional ✅
- Well structured files ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**